_docs/technical/2017-02-16-delegation.md

```diff
@@ -93,5 +100,34 @@ given issuer's public key, signature and message itself.
 ### Confirmation of proxy signature delivery
 
 Delegate should take the proxy signing key he has and sign this key with itself.
-If the signature is correct, then it was done by delegate (guaranteed by PSK
-scheme).
+If the signature is correct, then it was done by delegate (guaranteed by PSK scheme).
```

Comment:
Ok, misunderstood
Need only to correct ~sign this key with itself~ to *make a signature of psk using psk and delegate's key* or something like that